gcloud/logging/logger.py

```diff
@@ -298,7 +298,7 @@ def list_entries(self, projects=None, filter_=None, order_by=None,
                   more entries can be retrieved with another call (pass that
                   value as ``page_token``).
         """
-        log_filter = 'logName:%s' % (self.name,)
+        log_filter = 'logName=%s' % (self.full_name,)
```

Comment:
No, the original version doesn't work. I'm guessing the API changed underneath it. But logger.list_entries() does not work for me as is (would be curious if you tried now whether it works for you).

The original version actually should work since the fully qualified name should contain its own name and the colon operator is contains. However, = is better anyway since it's indexed, and the colon operator currently seems to be buggy.

I have been playing with the API and discovered the following things:

projectId can take either just a projectId or a list of projectIds. The docs say it should take a list.

If you don't give it a list, both = and : operator will work, but : operator will return an extraneous token, that if used will keep returning just itself.

If you do give it a list ,only = will work, : operator will return 0 results with a nextPageToken that keeps returning itself.

I am following up internally about these issues but I think even if these bugs are fixed, this is the correct change to make since it's indexed.